src/Tasks/Hash.cs

```diff
@@ -37,61 +35,102 @@ public class Hash : TaskExtension
         [Output]
         public string HashResult { get; set; }
 
+        private static readonly char ItemSeparatorCharacter = '\u2028';
```

Comment:
Please either use const and rename to `itemSeparatorCharacter` or rename it to `s_itemSeparatorCharacter` to follow naming rules for private statics and private const.